jquery-1.12.4.min.js on CDN is not compressed

was working on an older site and updated it to load jQuery 1.12.4 via CDN, but pagespeed dev insights says it isn't compressed. i then reviewed response headers and that does appear to be the case. screenshots below:

![image](https://github.com/jquery/codeorigin.jquery.com/assets/12283250/bbedbcf9-8e32-4bd4-a6a7-6db3f9f5e746)

![image](https://github.com/jquery/codeorigin.jquery.com/assets/12283250/8cf1d798-3489-4aae-ad75-08e9306f991a)

@Krinkle OK - i played around with this a bit, and thought i had a fairly reasonably modification to the test:

https://github.com/EvaLok/infrastructure-puppet/blob/d94418e0ba909a07f0680d15d58f2ad7d20c429d/test/CodeoriginTest.php#L28-L42

```php
Unit::testHttp( $server, '/jquery-3.0.0.js', [
	"Accept-Encoding: gzip, deflate, br, zstd",
], [
	'status' => '200',
	'server' => 'nginx',
	'content-type' => 'application/javascript; charset=utf-8',
	'content-encoding' => 'gzip',
	'content-length' => '77731',
	'last-modified' => 'Fri, 18 Oct 1991 12:00:00 GMT',
	'vary' => 'Accept-Encoding',
	'etag' => '"28feccc0-40464"',
	'cache-control' => 'public, max-age=31536000, stale-while-revalidate=604800',
	'access-control-allow-origin' => '*',
	'accept-ranges' => 'bytes',
] );
```

however, while most of the servers checked pass the test, the `codeorigin-02.stage.ops.jquery.net` server seems to gives an error:

https://github.com/EvaLok/infrastructure-puppet/actions/runs/9103199343/job/25024530795#step:3:23

```

Run make test
✅ test-codeorigin-next-http
✅ test-codeorigin-prod-http
✅ test-releases
not ok 8 GET https://codeorigin-02.stage.ops.jquery.net/jquery-3.0.0.js > header content-length
  ---
  actual:   null
  expected: "77731"
not ok 14 GET https://codeorigin-02.stage.ops.jquery.net/jquery-3.0.0.js > header accept-ranges
  ---
  actual:   null
  expected: "bytes"
✅ test-codeorigin-next-https
✅ test-codeorigin-prod-https
✅ test-contentorigin-prod
make: *** [Makefile:49: test-codeorigin-stage] Error 1
make: *** Waiting for unfinished jobs....
✅ test-miscweb
✅ test-wpdocs
```

is it expected that this server will not support the `Accept-Encoding` request header, and if so, how do you propose to modify the test so that we are able to make this assertion for other servers?

--- a/test/CodeoriginTest.php
+++ b/test/CodeoriginTest.php
@@ -29,6 +29,7 @@ Unit::testHttp( $server, '/jquery-3.0.0.js', [], [
 	'status' => '200',
 	'server' => 'nginx',
 	'content-type' => 'application/javascript; charset=utf-8',
+	'content-encoding' => 'gzip',
 	'content-length' => '263268',
 	'last-modified' => 'Fri, 18 Oct 1991 12:00:00 GMT',
 	'vary' => 'Accept-Encoding',

--- a/test/CodeoriginTest.php
+++ b/test/CodeoriginTest.php
@@ -26,7 +26,7 @@ Unit::testHttp( $server, '/', [], [
 // Static assets
 
 Unit::testHttp( $server, '/jquery-3.0.0.js', [
-	'accept-encoding' => 'gzip, deflate, br, zstd',
+	"Accept-Encoding: gzip, deflate, br, zstd",
 ], [
 	'status' => '200',
 	'server' => 'nginx',

--- a/test/CodeoriginTest.php
+++ b/test/CodeoriginTest.php
@@ -32,7 +32,7 @@ Unit::testHttp( $server, '/jquery-3.0.0.js', [
 	'server' => 'nginx',
 	'content-type' => 'application/javascript; charset=utf-8',
 	'content-encoding' => 'gzip',
-	'content-length' => '263268',
+	'content-length' => '77731',
 	'last-modified' => 'Fri, 18 Oct 1991 12:00:00 GMT',
 	'vary' => 'Accept-Encoding',
 	'etag' => '"28feccc0-40464"',

--- a/test/CodeoriginTest.php
+++ b/test/CodeoriginTest.php
@@ -108,6 +108,20 @@ Unit::testHttp( $server, '/jquery-3.0.0.js', [
 	'access-control-allow-origin' => '*',
 ] );
 
+Unit::testHttp( $server, '/qunit/qunit-2.0.0.css', [
+	"Accept-Encoding: gzip, deflate, br, zstd",
+], [
+	'status' => '200',
+	'server' => 'nginx',
+	'content-type' => 'text/css',
+	'content-encoding' => 'gzip',
+	'last-modified' => 'Fri, 18 Oct 1991 12:00:00 GMT',
+	'vary' => 'Accept-Encoding',
+	'etag' => '"28feccc0-1d20"',
+	'cache-control' => 'public, max-age=31536000, stale-while-revalidate=604800',
+	'access-control-allow-origin' => '*',
+] );
+
 // Renamed files
 
 Unit::testHttp( $server, '/jquery-git2.js', [], [

--- a/test/CodeoriginTest.php
+++ b/test/CodeoriginTest.php
@@ -25,7 +25,9 @@ Unit::testHttp( $server, '/', [], [
 
 // Static assets
 
-Unit::testHttp( $server, '/jquery-3.0.0.js', [], [
+Unit::testHttp( $server, '/jquery-3.0.0.js', [
+	'accept-encoding' => 'gzip, deflate, br, zstd',
+], [
 	'status' => '200',
 	'server' => 'nginx',
 	'content-type' => 'application/javascript; charset=utf-8',

--- a/test/CodeoriginTest.php
+++ b/test/CodeoriginTest.php
@@ -25,14 +25,11 @@ Unit::testHttp( $server, '/', [], [
 
 // Static assets
 
-Unit::testHttp( $server, '/jquery-3.0.0.js', [
-	"Accept-Encoding: gzip, deflate, br, zstd",
-], [
+Unit::testHttp( $server, '/jquery-3.0.0.js', [], [
 	'status' => '200',
 	'server' => 'nginx',
 	'content-type' => 'application/javascript; charset=utf-8',
-	'content-encoding' => 'gzip',
-	'content-length' => '77731',
+	'content-length' => '263268',
 	'last-modified' => 'Fri, 18 Oct 1991 12:00:00 GMT',
 	'vary' => 'Accept-Encoding',
 	'etag' => '"28feccc0-40464"',
@@ -93,6 +90,22 @@ Unit::testHttp( $server, '/jquery-3.0.0.js', [
 	'cache-control' => 'public, max-age=31536000, stale-while-revalidate=604800',
 	'access-control-allow-origin' => '*',
 	'accept-ranges' => 'bytes',
+] );
+
+// Gzip Compression
+
+Unit::testHttp( $server, '/jquery-3.0.0.js', [
+	"Accept-Encoding: gzip, deflate, br, zstd",
+], [
+	'status' => '200',
+	'server' => 'nginx',
+	'content-type' => 'application/javascript; charset=utf-8',
+	'content-encoding' => 'gzip',
+	'last-modified' => 'Fri, 18 Oct 1991 12:00:00 GMT',
+	'vary' => 'Accept-Encoding',
+	'etag' => '"28feccc0-40464"',
+	'cache-control' => 'public, max-age=31536000, stale-while-revalidate=604800',
+	'access-control-allow-origin' => '*',
 ] );
 
 // Renamed files

